recipes/emboss/5.0.0/meta.yaml

```diff
@@ -4,6 +4,7 @@ package:
 
 build:
   number: 0
+  noarch: generic
```

Comment:
From the linter messages: "The recipe uses per platform sources and cannot be noarch". We need to somehow skip the check for noarch, but this is wrong I think.